app/services/random_phrase_service.rb

```diff
@@ -0,0 +1,14 @@
+class RandomPhrase 
+    def generate_phrase(type)
+        response = conn.get("api/v1/#{type}")
+        parse_json(response)
+    end
+
+    def self.conn 
+        Faraday.new('https://shielded-headland-00998.herokuapp.com')
+    end
+
+    def self.parse_json(response)
```

Comment:
we can make a Module for #methods like these which are shared across multiple classes